src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -48,6 +49,7 @@
   private String defaultGcsBucketName;
   private Boolean clearDatastore;
   private File datastorePath;
+  private Map<String, String> environment;
```

Comment:
I don't actually know the behavior, but I believe the env_var flag was specifically to do something that would allow them to be readable by the servers. Perhaps @davidjn might know though.